src/TeleSign.Services/TeleSignService.cs

```diff
@@ -52,6 +52,7 @@ public class TeleSignService
                     IWebRequester webRequester,
                     string accountName = "default")
         {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
I would be careful setting this from within the library.  While it makes sense to use TLS, this setting is shared across the app domain and setting it here will change the value for all requests which may be an unexpected behavior since it's wouldn't be scoped to just TeleSign calls.